### PR TITLE
chore(ci): add firefox to test matrix and use more compact reporter

### PR DESCRIPTION
**Pull request by anehx**
Running tests now looks like this:

Selection_019

Failed tests would be below the dot matrix with a clear error

### CODE REVIEW

File: testem.js [hunk 1]
```diff
@@ -16,6 +19,12 @@ module.exports = {
         "--remote-debugging-port=0",
         "--window-size=1440,900"
       ].filter(Boolean)
+    },
+    Firefox: {
+      ci: [
+        "--headless",
```
Comment (fkm) [inline testem.js]: Did you test this / does this work?

From the documentation the headless flag has only one dash:
https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Headless_mode

Comment (anehx) [reply testem.js]: Interesting, yes I tested it and yes it works. However, you're right I'll change it to match the docs

Comment (anehx) [reply testem.js]: Done

Comment (fkm) [reply testem.js]: Good to know! :-)

**Pull request merged**